ftxwebsocket/client.py

```diff
@@ -13,15 +13,14 @@
 class FtxWebsocketClient(WebsocketManager):
     _ENDPOINT = 'wss://ftx.com/ws/'
 
-    def __init__(self, on_disconnect: callable = None) -> None:
```

Comment:
Relative to part 1, removed the `on_disconnect` callback. This was used in part 1 for triggering RESTful fetching of missed trade records during websocket available, but it is not necessary in part 2 because the goal is no longer to print ordered trades, but, rather, to compute an index price based on the *latest* trades on each market. We reason that the websocket will reconnect at a comparable rate to the restoration of REST request availability, so there does not appear to be a need for triggering REST callbacks.